src/components/HeaderBar.vue

```diff
@@ -0,0 +1,96 @@
+<script setup>
+import IconBaseball from '../components/icons/IconBaseball.vue';
+import IconBasketball from '../components/icons/IconBasketball.vue';
+import IconFootball from '../components/icons/IconFootball.vue';
+import IconHockey from '../components/icons/IconHockey.vue';
+import { ref } from 'vue';
+
+const props = defineProps({ allSports: { type: Array, required: true } })
+// let thisComponent = ref(IconBaseball)
+// ^^^tried using this to make the component :is dynamic, but will circle back to it
+</script>
+
+<template>
+  <div class="container">
+    <div class="sports-wrapper">
+      <button
+        v-for="sport in allSports" 
+        :key="sport.key"
+        class="sport"
+      >
+        <IconBaseball
+          v-if="sport.key === 'mlb'"
+          class="icon"
+        />
+      
+        <IconHockey
+          v-else-if="sport.key === 'nhl'" 
+          class="icon"
+        />
+      
+        <IconBasketball
+          v-else-if="sport.key === 'nba'" 
+          class="icon"
+        />
+      
+        <IconFootball
+          v-else-if="sport.key === 'nfl'" 
+          class="icon"
+        />
+      
+        <p class="button-text">
+          {{ sport.key.toUpperCase() }}
+        </p>
+      </button>
+    </div>
+  </div>
+</template>
+
+<style>
+.container {
+  width: 100%;
+}
+
+.sports-wrapper {
+  height: 50px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.sport {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transform: skew(-30deg);
+  padding-left: 6VW;
+  padding-right: 6VW;
+  outline: none;
```

Comment:
ok sweet. i used em for everything in my capstone and had no idea what it was haha, also now I see rem wouldve been super helpful and I think that's why weird stuff was happening where I'd have to like double the em because it wasn't acting the way I was expecting.

Reply:
trying to get it to react with em, but not much success. Going to skip for now